src/main/scala/com/cognite/sdk/scala/v1/timeSeries.scala

```diff
@@ -57,7 +57,16 @@ final case class TimeSeriesSearchFilter(
 )
 
 final case class TimeSeriesFilter(
-    assetIds: Option[Seq[Long]] = None
+    name: Option[String] = None,
+    unit: Option[String] = None,
+    isString: Option[Boolean] = None,
+    isStep: Option[Boolean] = None,
+    metadata: Option[Map[String, String]] = None,
+    assetIds: Option[Seq[Long]] = None,
+    rootAssetIds: Option[Seq[Long]] = None,
```

Comment:
So this should be `Long` not `CogniteId`?

Reply:
That's right!